packages/stark-ui/src/modules/session-ui/session-ui.module.ts

```diff
@@ -1,50 +1,32 @@
-import { ApplicationInitStatus, Inject, ModuleWithProviders, NgModule, Optional, SkipSelf } from "@angular/core";
+import { ModuleWithProviders, NgModule, Optional, SkipSelf } from "@angular/core";
 import { UIRouterModule } from "@uirouter/angular";
 import { TranslateModule } from "@ngx-translate/core";
-import { CommonModule, Location } from "@angular/common";
+import { CommonModule } from "@angular/common";
 import { MatButtonModule } from "@angular/material/button";
-import { SESSION_MODULE_STATES } from "./routes";
-import { from } from "rxjs";
-import {
-	STARK_ROUTING_SERVICE,
-	STARK_SESSION_CONFIG,
-	StarkSessionConfig,
-	starkLoginStateName,
-	starkPreloadingStateName,
-	StarkRoutingService
-} from "@nationalbankbelgium/stark-core";
+import { SESSION_UI_STATES } from "./routes";
 import {
 	StarkLoginPageComponent,
 	StarkPreloadingPageComponent,
 	StarkSessionExpiredPageComponent,
 	StarkSessionLogoutPageComponent
 } from "./pages";
-import { StarkAppContainerComponent } from "./components";
 
 @NgModule({
 	declarations: [
-		StarkAppContainerComponent,
-		StarkLoginPageComponent,
-		StarkPreloadingPageComponent,
-		StarkSessionExpiredPageComponent,
-		StarkSessionLogoutPageComponent
-	],
-	exports: [
-		StarkAppContainerComponent,
 		StarkLoginPageComponent,
 		StarkPreloadingPageComponent,
 		StarkSessionExpiredPageComponent,
 		StarkSessionLogoutPageComponent
 	],
+	exports: [StarkLoginPageComponent, StarkPreloadingPageComponent, StarkSessionExpiredPageComponent, StarkSessionLogoutPageComponent],
```

Comment:
Has Prettier been runned on this file ? Could you reformat this ? 😊

Reply:
Yes, in fact it was `prettier` the one that changed this line ;)